tests/BUILD

```diff
@@ -1,6 +1,11 @@
 cc_test(
     name = "simulator-test",
-    srcs = ["simulator_test.cpp"],
+    srcs = [
+        "ccnode_test.cpp",
```

Comment:
nit: It's standard to put unit tests in the same directory as the source files they test for c++

Reply:
Done.